otp/build.gradle

```diff
@@ -1,4 +1,32 @@
 description = 'OTP (One Time Password) utility in Java'
+configurations.all {
```

Comment:
As discussed & shown please keep the config in Root Build.gradle [all projects section]